WeakAurasTemplates/TriggerTemplates.lua

```diff
@@ -1179,6 +1281,62 @@ function WeakAuras.CreateTemplateView(frame)
     end
   end
 
+  local batchModeLabel = CreateFrame("FRAME", "batchModeLabel", newView.frame);
+  batchModeLabel:SetSize(300, 20);
+  local batchModeLabelString = batchModeLabel:CreateFontString(nil, "ARTWORK");
+  batchModeLabelString:SetFont(STANDARD_TEXT_FONT, 10); -- "OUTLINE"
+  batchModeLabelString:SetTextColor(1,1,1,1);
+  batchModeLabelString:SetText('|TInterface\\OptionsFrame\\UI-OptionsFrame-NewFeatureIcon:0|t' .. L["Hold CTRL to make multiple auras at once."]);
```

Comment:
"Hold CTRL to create multiple auras at once"

Reply:
amended with "create"